_jdg-quickstarts

```diff
@@ -1 +1 @@
-Subproject commit 7be4b6dfe68cb0d5b623528af18d0bc39471552f
+Subproject commit db2f24eeb88d165544195f4a0696bb82a6a531b6
```

Comment:
This links to JDG_6.5.1.CR1. You should link to the QS tag that matches the latest GA download available on the site. in this case 6.5.0.GA, which doesn't seem to be tagged in: https://github.com/jboss-developer/jboss-jdg-quickstarts/releases. You'll need to track down what happened there.